php/blocks/class-loader.php

```diff
@@ -332,8 +330,8 @@ public function render_block_template( $block, $attributes ) {
 			$this->enqueue_global_styles();
 		}
 
-		$block_lab_attributes = $attributes;
-		$block_lab_config     = $block;
+		block_lab()->data['attributes'] = $attributes;
+		block_lab()->data['config']     = $block;
```

Comment:
What do you think about storing `->data` as a protected property of this `Loader` class? 

That way, it wouldn't be possible to change the data outside this class.

Then, `helpers.php` could access the data with something like `block_lab()->loader->get_data( 'attributes' )`, or whatever method name.

#440 has infrastructure for this, including storing `Loader` as a property of `Plugin`, not in the protected property `->components`.

Reply:
If it's stored in the `Plugin` class, it has to be publicly accessible. But if it's stored in `Loader`, it can be set inside that class, and no other class can set it. They can only get the value.